tests/population/test_add_new_birth_cohort.py

```diff
@@ -58,10 +58,9 @@ def test_FertilityDeterministic(config):
 
     components = [TestPopulation(), FertilityDeterministic()]
     simulation = InteractiveContext(components=components, configuration=config)
-    num_steps = simulation.run_for(duration=pd.Timedelta(days=num_days))
+    simulation.run_for(duration=pd.Timedelta(days=num_days))
```

Comment:
This update stems from this mypy pr -> https://github.com/ihmeuw/vivarium/pull/566. The assertion seems unnecessary here as it is a core fundamental of vivarium and not the component being tested.